ros_ws/src/navigation_stack/msgs_converter/include/msgs_converter.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <drive_msgs/drive_param.h>
+#include <geometry_msgs/Twist.h>
+
+class MSGSConverter
```

Comment:
Can this be renamed to avoid the acronym? Either way, a summary would be helpful here to explain what the class does and why it's needed.